app/utils/sentry.js

```diff
@@ -14,13 +14,13 @@ function initialiseSentry () {
   return Sentry
 }
 
-const addSentryToErrorLevel = (originalLogger) => {
+const addSentryToErrorLevel = originalLogger => {
   const sentryLogger = Object.create(originalLogger)
-  sentryLogger.error = msg => {
+  sentryLogger.error = function () {
     try {
-      Sentry.captureException(new Error(msg))
+      Sentry.captureException(new Error(JSON.stringify(arguments)))
```

Comment:
where does the `arguments` come from?

Reply:
its a JS thing
"arguments is an Array-like object accessible inside functions that contains the values of the arguments passed to that function."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments